src/pages/home/report/ReportActionItem.tsx

```diff
@@ -901,7 +932,12 @@ function ReportActionItem({
             >
                 {(hovered) => (
                     <View style={highlightedBackgroundColorIfNeeded}>
-                        {shouldDisplayNewMarker && <UnreadActionIndicator reportActionID={action.reportActionID} />}
+                        {shouldDisplayNewMarker &&
```

Comment:
```suggestion
                        {shouldDisplayNewMarker && (
    (!usedThreadDividerLine || !isFirstVisibleReportActionID) && 
    <UnreadActionIndicator reportActionID={action.reportActionID} />
)}
```

For more readable

Reply:
@dragnoir How does this change logic ??

<img width="718" alt="Screenshot 2024-04-26 at 16 41 25" src="https://github.com/Expensify/App/assets/141406735/bcc6ccaf-927e-4b69-9434-bdd9770e3363">

Reply:
@dragnoir How about this point? I complete the checklist. Waiting for this update to approve